### ISSUE TITLE
Design Issue [MEDIUM] : allow existing user login on "request a loan" modal popup

### ISSUE THREAD

**Issue by mzaradzki** [JulienPageaud/mobile-data-scoring-app#308]
Problem : 
The "request a loan" button is central to the page, but it asks users to create a new account.
Existing users needs to first login with the top-right corner button to use this button.

Change to do : 
The modal popup should offer two choices : (1) existing user login or (2) new user sign-up

**Comment by cdimitroulas** [comment]
Can we just add a login link under the request a loan button?

Like on most website there is a link with the text "Already have an account? Sign in here!"

**Comment by mzaradzki** [comment]
![capture d ecran 2017-01-28 a 15 33 33](https://cloud.githubusercontent.com/assets/13558380/22397323/42e036c8-e56f-11e6-92c0-aa0fef9fa1a6.png)

I think it is best to assume the user already exists and to display a message like the one on hotmail (screenshot) saying : "dont have an account ? create a Stride profile now"

Note : this is a Design Enhancement only (not High or Medium priority)

**Comment by JulienPageaud** [comment]
Yes, you are right it is better to consider the user has already signed up
